build-system/tasks/helpers.js

```diff
@@ -150,7 +150,22 @@ async function bootstrapThirdPartyFrames(options) {
  * @param {!Object} options
  */
 async function compileCoreRuntime(options) {
-  await doBuildJs(jsBundles, 'amp.js', options);
+  async function watchFunc() {
+    const bundleComplete = await doBuildJs(jsBundles, 'amp.js', {
+      ...options,
+      watch: false,
```

Comment:
I think the code as written will bail from the watcher and the running server the moment there's a compilation error in `amp.js`. If this is indeed the case, you can add `continueOnError: true` to this `options` object to prevent it.

Reply:
> I think the code as written will bail from the watcher and the running server the moment there's a compilation error in amp.js. If this is indeed the case, you can add continueOnError: true to this options object to prevent it.

From testing, errors in compiling `amp.js` didn't quit the program